great_expectations/render/renderer/content_block/validation_results_table_content_block.py

```diff
@@ -92,20 +92,29 @@ def _process_content_block(
 
             content_block.styling = styling
 
+    @override
     @classmethod
-    def _get_content_block_fn(cls, expectation_type):
-        expectation_string_fn = get_renderer_impl(
-            object_name=expectation_type, renderer_type=LegacyRendererType.PRESCRIPTIVE
+    def _get_content_block_fn(  # noqa: PLR0915
+        cls,
+        expectation_type: str,
+        expectation_config: ExpectationConfiguration | None = None,
+    ) -> Callable | None:
+        content_block_fn = cls._get_content_block_fn_from_expectation_description(
```

Comment:
This method is identical to the one it overrides with some extra stuff at the bottom. I know it has an early return in it that makes it harder but it would be nice to call the super version and then do the other stuff if possible.